.github/workflows/flutter.yml

```diff
@@ -77,7 +77,7 @@ jobs:
             flutter pub get
 
         - name: Test chrome
-          if: runner.os != 'macOS'
+          if: runner.os == 'Linux'
```

Comment:
https://github.com/getsentry/sentry-dart/issues/500
Right now it only happens on Windows, let's see if that helps.